### PR TITLE
Fw-Visual-Regression Crayons WC

### PR DESCRIPTION
**Pull request by detiwari003**
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)

### CODE REVIEW

**Review by parsuram-vijaysankar — CHANGES_REQUESTED** [review]
- Better you base your branch on the latest conary and raise a new PR, lots of changes seem to be there that might be old. 
- The title of the PR should be semantically named, check contribution guideline. 
- Remove the Reddit references.
- We should not be committing the Report and base Screenshots! 
    - Add a gitignore so that future developers don’t commit these files.
- Able to see tests only for Dropdown and Data-table, where are the tests for Select dropdown and other components?
- Let’s remove the Clone TestDatePicker!

File: package.json [hunk 1]
```diff
@@ -27,7 +27,10 @@
     "release:ci": "HUSKY_INSTALL=1 HUSKY_SKIP_HOOKS=1 lerna publish from-package --no-commit-hooks --dist-tag canary --yes",
     "storybook:build": "rimraf storybook-dist && build-storybook -c docs/.storybook -o docs/storybook-dist",
     "storybook:dev": "lerna run build --stream && start-storybook -p 9001 -c docs/.storybook",
+    "storybook:test":"start-storybook -p 9001 -c docs/.storybook --ci",
     "test": "lerna run test --stream",
+    "visual-test": "start-server-and-test 'npm run storybook:test' 9001 'npm run exe-visual-test' ",
```
Comment (arvindanta) [inline package.json:32]: can you please order the scripts in alphabetical order ?

File: package.json [hunk 2]
```diff
@@ -65,14 +68,17 @@
     "kill-port": "^1.6.1",
     "lerna-clean-changelogs-cli": "^2.0.2",
     "lint-staged": "^10.5.4",
+    "lit": "^2.1.4",
```
Comment (arvindanta) [inline package.json:71]: do we need `lit` as dev dependency ?

File: packages/crayons-core/src/components/crayons-visual-regression-tests/readme.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+6. After running your tests , just go to `__crayonsDIR__/visual-regression-tests/jest-screenshot-report` path and open the report via `http-server`. You 
+   can see the regressions and correct them. 
+7. After correcting again run the same command with `jest -u`. `-u` option helps you to save new snapshots that will be used for further integrations.
+
+<!-- Auto Generated Below -->
+
+
+## Properties
+
+| Property | Attribute | Description     | Type     | Default     |
```
Comment (arvindanta) [inline packages/crayons-core/src/components/crayons-visual-regression-tests/readme.md:27]: Where are these properties being used ?. Can you re generate this readme file ?

**Review by arvindanta — CHANGES_REQUESTED** [review]

**Pull request closed without merging**